docs/src/main/asciidoc/deploying-to-google-cloud.adoc

```diff
@@ -0,0 +1,109 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Deploying on Google cloud platform
+
+include::./attributes.adoc[]
+
+This guide covers:
+
+* Connecting to a CloudSQL postgresql instance with quarkus-agroal and service account using CloudRun
+== Prerequisites
+
+For this guide you need:
+
+* roughly 1 hour
+* having access to an Google cloud platform project with owner rights.
+
+This guide will take as input an application developed in the link:datasource.adoc[datasource guide].
+
+Make sure you have the application at hand working locally.
+
+
+== Solution
+
+We recommend to follow the instructions in the next sections and build the application step by step.
+
+== Connecting to CloudSQL
+
+Google CloudSQL managed service allows 4 kinds of connection :
+
+. Using public IP
+. Using private IP
+. Using Cloud SQL Proxy
+. Using service account
+
+The first two don't need anymore details, simply connect to you database following the link:datasource.adoc[datasource guide].
```

Comment:
```suggestion
The first two don't need anymore details, simply connect to your database by following the link:datasource.adoc[datasource guide].
```

Reply:
> Added more comments. Will this work in native too?
To be honest. I compiled it in native mode. run the code. But I didn't actually connect to SQL from Cloud Run.